Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -474,38 +474,6 @@ object LocalData {
 
     fun submissionWasSuccessful(): Boolean = numberOfSuccessfulSubmissions() >= 1
 
-    fun testGUID(): String? = getSharedPreferenceInstance().getString(
-        CoronaWarnApplication.getAppContext()
-            .getString(R.string.preference_test_guid),
-        null
-    )
-
-    fun testGUID(value: String?) {
-        getSharedPreferenceInstance().edit(true) {
-            putString(
-                CoronaWarnApplication.getAppContext()
-                    .getString(R.string.preference_test_guid),
```

Comment:
If they are unused, can we remove them from the `values-de/strings.xml` such that are removed from all files on the next translation update?

Reply:
sure, will do :)